src/modules/ChatModule.ts

```diff
@@ -44,6 +45,20 @@ class ChatModule implements events.IStats {
         return this.messages
     }
 
+    toLogstf(): Chat[] {
+        const chat: Chat[] = []
+        for (const chatItem of this.messages) {
+            chat.push(<Chat>renameObjectKeys(chatItem, new Map<string, any>([
```

Comment:
Quick tip:

This is the old way to do a static cast, we should just do `new Map<string, Chat>`, or do a cast with `as`: `new Map<>() as Chat`